### PR TITLE
Add proposal for invoices.send

### PR DESCRIPTION
**Pull request by luisaterrabrito**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -2425,6 +2425,20 @@ Register a payment for an invoice.
...
 
+### invoices.send [POST /invoices.send]
+
+Send an invoice.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `7abb325c-e063-42a4-8fb4-1b730759645a` (string, required)
+        + to: `mary@teamleader.eu`, `john@teamleader.eu` (array[string], required)
```
Comment (dcsg) [inline apiary.apib:2436]: Q: Should we also include the `cc` and `bcc` in this proposal? In the UI is possible to set those as well.

File: apiary.apib [hunk 2]
```diff
@@ -2425,6 +2425,20 @@ Register a payment for an invoice.
...
+        + subject: `compliments` (string, required)
+        + content: `amazing compliments` (string, required)
```
Comment (dcsg) [inline apiary.apib:2438]: Q: would it also make sense to accept the template to be used for the email?

Comment (luisaterrabrito) [reply apiary.apib:2438]: that is for later

**Comment by luisaterrabrito** [comment]
Still to be discussed

**Pull request closed without merging**